src/main/java/com/lob/protocol/response/CheckResponse.java

```diff
@@ -95,8 +101,16 @@ public String getUrl() {
         return url;
     }
 
-    public TrackingResponse getTracking() {
-        return tracking;
+    public String getCarrier() {
+        return carrier;
+    }
+
+    public String getTrackingNumber() {
+        return trackingNumber;
+    }
+
+    public List<TrackingEventResponse> getTrackingEvents() {
+        return trackingEvents;
```

Comment:
@robinjoseph08 In `LetterResponse` and `PostcardResponse`, thumbnails is returned directly. Is there a reason for this or should I change `getThumbnails` in those files?